server/ai/configuration_test.go

```diff
@@ -0,0 +1,262 @@
+package ai
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBotConfig_IsValid(t *testing.T) {
```

Comment:
There isn't a success case. So a function that always fails will pass. (Which is the current case)